README.md

```diff
@@ -170,3 +170,110 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+#Deployment to Fly.io
+
+
+## Install FlyCtl
+
+To work with the Fly platform, you first need to install Flyctl, 
+a command line interface that allows tou to do everything from creating an account to deploy to Fly.
+
+  - To install it on Linux
+
+    ```  
+      curl  -L https:fly.io/install.sh | sh`
+    ```
+
+- To install it on windows
+    ```
+      iwr https://fly.io/install.ps1 -useb
+    ```
+    
+    Visit `https://fly.io/` if you need installation guide
+  
+
+- Next authenticate with your fly.io account:
+    ```
+     fly auth login`
+    ```
+
+
+- To make sure everything is working well:
+    ```
+     fly apps list
+    ```
+  The output of the above command will be empty table since you have no apps launched yet
+
+
+## Configure the Project
+
+  Environment Variables
+
+  We shouldn't store secrets in source code, so utilizing environmental variables is needed.
+
+  Generate SECRET_KEY (if change needed)
+  - Run the following from the root of your project
+    ```
+    python3 generate_key.py
+    ```
+    Get your key and update in .env file SECRET_KEY='<your-key>'
```

Comment:
What?

As I said. `generate_key.py` should check if .env exists. If not, add this key. Otherwise, abort with an error.

MINIMIZE THE AMOUNT OF MANUAL WORK

Reply:
It's not even obvious how to add a secret key that's multiline. Do it with code

Reply:
I removed this one since it's generated no need to generate again and if so can use random string.